setup.py

```diff
@@ -0,0 +1,36 @@
+import sys
+
+from setuptools import setup, find_packages
+
+from baseplate.integration.thrift.command import ThriftBuildPyCommand
+
+
+PY3 = (sys.version_info.major == 3)
+
+tests_require = [
+    "nose",
+    "coverage",
+    "webtest",
+]
+
+if not PY3:
+    tests_require.append("mock")
+
+setup(
+    name="reddit_service_ads_tracking",
+    packages=find_packages(),
+    install_requires=[
+        "pyramid",
+        "baseplate",
+    ],
+    cmdclass={
+        "build_py": ThriftBuildPyCommand,
+    },
+    entry_points={
+        "paste.app_factory": [
```

Comment:
not really sure, was having trouble getting it to run and adding this seemed to work.

to be clear i can remove the `entry_points` section and just add the follow to the ini?
```
[app:reddit_service_ads_tracking]
use = egg:baseplate
```